applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -598,6 +598,35 @@ public function post_requestPassword(array $body) {
         $this->validateModel($this->userModel, true);
     }
 
+    /**
+     * @param $id
```

Comment:
Need some more detail in this doc block. Explain what we're doing here. Add brief descriptions to the parameters (you can copy from other methods in the class).